File: BBIS.Application/DTOs/Common/PagedSearchDto.cs
namespace BBIS.Application.DTOs.Common
{
    public class PagedSearchDto
    {
        public string SearchText { get; set; }
        public int PageNumber { get; set; }
        public int PageSize { get; set; }
        public string SortBy { get; set; }
        public bool SortDesc { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/PagedSearchDto.ts
export class PagedSearchDto {
  SearchText: string = '';
  PageNumber: number = 1;
  PageSize: number = 10;
  SortBy: string = '';
  SortDesc: boolean = false;
}

export class PagedSearchResultDto<T> {
  TotalCount: number = 0;
  CurrentPage: number = 0;
  PageSize: number = 0
  Results: Array<T> = [];
}